### ISSUE TITLE
summary and print for model with dispformula

### ISSUE THREAD

**Issue by mebrooks** [glmmTMB/glmmTMB#153]
In models with a non-default `dispformula`, the `summary` function is expecting `vcov` to return a list with an element for the dispersion part of the model. It currently doesn't, and this causes problems for `mkCoeftab` 
```
> data(sleepstudy, package="lme4")
> fm1 <- glmmTMB(Reaction ~ Days +     (1|Subject), dispformula=~ Days, sleepstudy)
> summary(fm1)
Error in coefs[, 2] : subscript out of bounds
In addition: Warning message:
In cbind(Estimate = coefs, `Std. Error` = sqrt(diag(vcov))) :
  number of rows of result is not a multiple of vector length (arg 1)
```
The print function also returns an error, but I haven't tracked it down yet.
```
> fm1
Formula:          Reaction ~ Days + (1 | Subject)
Dispersion:                ~ Days
   Data: sleepstudy
      AIC       BIC    logLik  df.resid 
1793.8238 1809.7885 -891.9119       175 
Random-effects (co)variances:

Conditional model:
Error in reMat[, "Name"] <- c(unlist(lapply(varcor, colnames)), if (useScale) "") : 
  number of items to replace is not a multiple of replacement length
```
I might get back to this, but feel free to jump in if you know the solution.

**Comment by bbolker** [comment]
I think this is fixed by 0315834 , but have a look ... there are some minor annoyances with handling sets of dispersion parameters that have more than one element (i.e., because `lme4::formatVC` expects that the `sc` attribute of a VarCorr object has length 1 ...)

**Comment by mebrooks** [comment]
Yep, that fixed the problem. Thanks!
